### PR TITLE
added below the belt feat

### PR DESCRIPTION
**Pull request by bleehu**
modified:   03_Feats.txt
	modified:   ../Combat Rules/2_Shooting.txt

A reduction from -3 to -2 might not sound like a big deal, but it can be combined with several attachments to get fairly scary hipfire synergies. The range limit on it I think balances that out well, and the CQB qualifier means you can't hip-fire well with an assault rifle, chain gun, or rocket launcher. Looking for sanity check and push back. Issue #507

### CODE REVIEW

**Review by Reveraine — APPROVED** [review]
I like this feat.

**Comment by Turtlelord26** [comment]
If Ralaak takes this feat, he has no penalty for hip firing half the time (in r1)
In depth review when I get home

**Comment by bleehu** [comment]
No penalty for firing an SMG single-fire at point-blank range. I think that's balanced.

File: CharacterCreation/03_Feats.txt
Comment (rosebrallier) [inline CharacterCreation/03_Feats.txt]: "Combat Rules" (capitalized since it's a document title)

**Review by rosebrallier — APPROVED** [review]
One grammar change. Otherwise I say shipit pending @Turtlelord26 approval

File: CharacterCreation/03_Feats.txt
Comment (rosebrallier) [inline CharacterCreation/03_Feats.txt]: "in the Combat"

**Comment by Turtlelord26** [comment]
There are several SMGs with 4 (-1 per 5 DEX) auto accuracy, most of which all shoot 6 bullets per action. AS an example, the lv 10 laser smg: https://github.com/bleehu/Compound_X/blob/dev/Items/04_03_SMGs_%26_Shotguns.txt#L144
If I have 5 dex, it autofires with -3 accuracy.
Add a lv 10 attachment called Heavy Recoil Compensator and a Grip-pod and I'm down to minus 0 accuracy during autofire.
Or at level 1, an angled foregrip plus a heavy profile receiver will do the trick as well.


This may be more of a comment on the autofire penalty `+4 - 1 for every 5 Dex` or on how several different kinds of attachment can buff autofire accuracy.
I'm inclined to let the feat in for now and see if Ralaak breaks it in half.

However, we've still got our CQB carbine sneaking in.
https://github.com/bleehu/Compound_X/blob/dev/Items/04_04_Carbines_AssaultRifles_%26_BattleRifles.txt#L82

**Comment by bleehu** [comment]
I want to capture some notes about various "hip fire builds" in design before I merge this. I think you've shown we've come right up to the edge of being Iver powered. It creates a pitfall for future design.

**Pull request merged**
